src/js/components/RangeInput/stories/Disabled.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Box, Grommet, RangeInput } from 'grommet';
+
+export const Disabled = () => {
+  const [value, setValue] = React.useState(5);
+
+  const onChange = (event) => setValue(event.target.value);
+
+  return (
+    <Grommet
```

Comment:
I think there are enough stories demoing this behavior, can we leave this story as it was?